sites/offshore-mag/server/routes/website-section.js

```diff
@@ -1,8 +1,14 @@
 const queryFragment = require('@endeavorb2b/base-website-themes/pennwell/api/fragments/website-section-page');
 const { withWebsiteSection } = require('@base-cms/marko-web/middleware');
 const section = require('../templates/website-section');
+const mapspostersTemplate = require('../templates/website-section/resources-maps-posters');
 
 module.exports = (app) => {
+  app.get('/:alias(resources/maps-posters)', withWebsiteSection({
+    template: mapspostersTemplate,
```

Comment:
Proper camelcasing please, `mapsPostersTemplate`.

Reply:
Got ya, I updated that.